### DISCUSSION TITLE
Creating a compiled exe-program based on Mtex and Matlab

### DISCUSSION THREAD

**Discussion by gendalfini** [mtex-toolbox/mtex#1953] [Ask Anything]
Hi,

I'm writing a program in Matlab using MTEX and I want to compile it into an exe file that will run on another PC without Matlab. However, when I run the compiled file, errors keep appearing when running Mtex. When I run the compiled program, Mtex starts (if I separately specified startup_mtex in the Matlab code) and begins to copy its files to temporary folders, while constantly losing different files and giving an error, or (if I specified install_mtex in the code) - mtex begins to add its own files to the matlab path and this causes a critical error.

How can I create a compiled application in Matlab that will link mtex files from the specified folder and work normally?

Thnak you.

**Comment by ralfHielscher** [comment]
I guess if you want to generate an MTEX exe file you need to replace startup_mtex by a method that just adds the correct folders to the search path, everything else what this function does seems not to be necessary.

Ralf.
